projects/extension/src/containers/Popup.tsx

```diff
@@ -1,23 +1,46 @@
 import React, { FunctionComponent, useEffect, useRef, useState } from "react"
 import * as material from "@material-ui/core"
 import GlobalFonts from "../fonts/fonts"
+import { makeStyles } from "@material-ui/core/styles"
 import { light, MenuButton, Tab, Logo } from "../components"
 import CallMadeIcon from "@material-ui/icons/CallMade"
 import { Background } from "../background/"
 import { TabInterface } from "../types"
+import { Tooltip } from "@material-ui/core"
+import ErrorIcon from "@material-ui/icons/Error"
+import ReplayIcon from "@material-ui/icons/Replay"
 
 const { createTheme, ThemeProvider, Box, Divider } = material
 
+const useStyles = makeStyles(() => ({
+  flexBetween: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  replayIcon: {
+    width: "1.2rem",
+    marginLeft: "0.2rem",
+    color: "green",
+    cursor: "pointer",
+  },
+  errorIcon: { width: "1.2rem", color: "red", cursor: "pointer" },
+}))
+
 const Popup: FunctionComponent = () => {
+  const classes = useStyles()
   const disconnectTabRef = useRef<(tapId: number) => void>((_: number) => {})
   const [activeTab, setActiveTab] = useState<TabInterface | undefined>()
   const [otherTabs, setOtherTabs] = useState<TabInterface[]>([])
+  const [crashError, setCrashError] = useState<string>("")
   const appliedTheme = createTheme(light)
 
   useEffect(() => {
     let isActive = true
     let unsubscribe = () => {}
 
+    chrome.storage.local.get(["crashError"], (res) => {
+      setCrashError(res.crashError)
+    })
```

Comment:
Does this auto-update if you call `chrome.storage.local.set(["crashError"], ...)`? I guess no?

Reply:
This updates every time the popup opens.If popup is open and the localStorage update then no - but based on the nature of the popup thats really an edge scenario